modules/otel-troubleshoot-metrics.adoc

```diff
@@ -19,26 +19,59 @@ The Operator creates a `<cr_name>-collector-monitoring` telemetry service that y
 
 .Procedure
 
-. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource:
+. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource (CR):
 
 +
 [source,yaml]
 ----
+# ...
   config: |
     service:
       telemetry:
         metrics:
           address: ":8888" # <1>
+# ...
 ----
 <1> The address at which the internal collector metrics are exposed. Defaults to `:8888`.
 
-// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
-
 . Retrieve the metrics by running the following command, which uses the port-forwarding Collector pod:
 +
 [source,terminal]
 ----
 $ oc port-forward <collector_pod>
 ----
 
-. Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors` depending on the deployment mode of the OpenTelemetry Collector:
+:
```

Comment:
This colon needs to be deleted, since it's rendering in the preview as a double colon at the end of the sentence above
```suggestion
```

Reply:
Thank you for spotting this.